_includes/header.html

```diff
@@ -1,118 +1,67 @@
+          <a class="usa-nav__link {% if page.url contains '/who-uses-login/' %}usa-current{% endif %}" href="{{ '/who-uses-login' | relative_url }}">
             <span>{% t nav.who_uses_login_gov %}</span>
           </a>
         </li>
         <li class="usa-nav__primary-item">
-          <a
-            class="usa-nav__link {% if page.url contains '/create-an-account/' %}usa-current{% endif %}"
-            href="{{ '/create-an-account' | relative_url }}"
-          >
+          <a class="usa-nav__link {% if page.url contains '/create-an-account/' %}usa-current{% endif %}" href="{{ '/create-an-account' | relative_url }}">
             <span>{% t nav.create_an_account %}</span>
           </a>
         </li>
         <li class="usa-nav__primary-item">
-          <a
-            class="usa-nav__link {% if page.url contains '/help/' %}usa-current{% endif %}"
-            href="{{ '/help' | relative_url }}"
-          >
+          <a class="usa-nav__link {% if page.url contains '/help/' %}usa-current{% endif %}" href="{{ '/help' | relative_url }}">
             <span>{% t nav.help_center %}</span>
           </a>
         </li>
       </ul>
-
-      <form
-        class="usa-search usa-search--small"
-        id="search_form"
-        action="https://search.usa.gov/search"
-        accept-charset="UTF-8"
-        method="get"
-      >
-        <input name="utf8" type="hidden" value="&#x2713;" />
-        <input
-          type="hidden"
-          name="affiliate"
-          id="affiliate"
-          value="login.gov"
-        />
-        <div role="search">
-          <input
-            type="search"
-            name="query"
-            id="query"
-            autocomplete="off"
-            class="usa-input"
-          />
-          <input
-            type="submit"
-            class="usa-button app-search"
-            name="commit"
-            value="{% t actions.search %}"
-            data-disable-with="Search"
-          />
-        </div>
-      </form>
+      {% include search.html size="small" %}
     </div>
 
     <div class="mobile btn-lang-toggle">
       {% include svg/globe.svg %}
-      <span class="label"> {% t global.language %} </span>
+      <span class="label">
+        {% t global.language %}
+      </span>
     </div>
 
     <ul class="mobile language-list">
       {% for locale in site.languages %}
-        {% capture name %}global.locales.{{locale }}{% endcapture %}
-      <li>
-        {% if locale == 'en' %}
-          <a href="{{ page.url | prepend: site.baseurl_root }}">{% t name %}</a>
-        {% elsif site.lang != 'en' and locale == site.lang %}
-          <a href="{{ page.url | prepend: site.baseurl  }}">{% t name %}</a>
-        {% else %}
-          <a
-            href="{{ page.url | prepend: locale | prepend: '/' | prepend: site.baseurl_root }}"
-            >{% t name %}</a
-          >
-        {% endif %}
-      </li>
+        {% capture name %}global.locales.{{locale }}
+        {% endcapture %}
```

Comment:
the spacing is confusing to me here, a few ideas:

1. balance the spacing around `locale`
2. bring `endcapture up`?

```suggestion
        {% capture name %}global.locales.{{ locale }}{% endcapture %}
```